src/java.desktop/share/classes/javax/swing/JTable.java

```diff
@@ -1164,7 +1164,7 @@ public Color getGridColor() {
      * @see     #setShowHorizontalLines
      */
     @BeanProperty(description
-            = "The color used to draw the grid lines.")
+            = "Whether grid lines should be drawn around the cells.")
```

Comment:
```suggestion
            = "Whether grid lines are drawn around the cells.")
```
Just use Present tense? This way it aligns with javadoc that uses, “the table *draws* grid lines…”